cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -120,15 +120,16 @@ type gceManagerImpl struct {
 	GkeService AutoscalingGkeClient
 	GceService AutoscalingGceClient
 
-	location              string
-	projectId             string
-	clusterName           string
-	mode                  GcpCloudProviderMode
-	templates             *templateBuilder
-	interrupt             chan struct{}
-	regional              bool
-	explicitlyConfigured  map[GceRef]bool
-	migAutoDiscoverySpecs []cloudprovider.MIGAutoDiscoveryConfig
+	location                 string
+	projectId                string
+	clusterName              string
+	mode                     GcpCloudProviderMode
+	templates                *templateBuilder
+	interrupt                chan struct{}
+	regional                 bool
+	explicitlyConfigured     map[GceRef]bool
+	migAutoDiscoverySpecs    []cloudprovider.MIGAutoDiscoveryConfig
+	machinesCacheLastRefresh time.Time
```

Comment:
nit: I would prefer if this was defined next to lastRefresh since the two are very similar and connected to cache.

Reply:
Done